### ISSUE TITLE
Make logs gitignore-able

### ISSUE THREAD

**Issue by Tallyb** [teambit/bit#2023]
When `bit build` failed, the following errors were created in the root folder with names similar to those ones: 

?? report.20190924.144738.77808.0.001.json
?? report.20190924.144929.77853.0.001.json
?? report.20190924.145017.77889.0.001.json

This pattern is not explicit enough to make the files added to gitignore. 

Files should end with log, so in most cases they are gitignored by default. Alternatively, they can be placed elsewhere and not in the project's root, so directory can be ignored.

**Comment by davidfirst** [comment]
@Tallyb , I believe these files were not created by Bit, probably by the compiler. If so, it'd be best to move the issue to bit.envs/envs repo.

**Comment by GiladShoham** [comment]
I'm pretty sure @davidfirst is right and it's not bit files.
Do you have a small repo to reproduce just to make sure?

**Comment by Tallyb** [comment]
https://docs.bit.dev/docs/tutorials/bit-vue-tutorial and start it: 
-clone repo
- add component
- add compiler
- run bit build. should work ok. 

Go to the productList.vue file and add inside the methods (e.g. line 36) and extra `}` so the file is faulty. 
Run bit build and you will get the log. 
note that if you run the native vue build, this file is not shown.

**Comment by GiladShoham** [comment]
@Tallyb I didn't got the files, but I notice few other issues:
1. When running the native vue build (npm run build), we got a normal error:
```

> bit-vue-tutorial@0.1.0 build /private/tmp/bit-vue-tutorial
> vue-cli-service build


⠦  Bu 98% after emitting CopyPlugin

 ERROR  Failed to compile with 1 errors                                                                                                                                 1:00:26 PM

 error  in ./src/components/ProductList.vue?vue&type=script&lang=js&

Syntax Error: SyntaxError: /private/tmp/bit-vue-tutorial/src/components/ProductList.vue: Unexpected token (37:0)

  35 |       }
  36 |   }}
> 37 | }
     | ^
  38 |


 @ ./src/components/ProductList.vue?vue&type=script&lang=js& 1:0-280 1:296-299 1:301-578 1:301-578
 @ ./src/components/ProductList.vue
 @ ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=script&lang=js&
 @ ./src/App.vue?vue&type=script&lang=js&
 @ ./src/App.vue
 @ ./src/main.js
 @ multi ./src/main.js
```
while bit gives some `JavaScript heap out of memory` error.

2. While debugging it, I saw that bit do accept the correct error after a second, but it keeps running for about 10 more seconds, so there is some memory leak during the process.

**Comment by imsnif** [comment]
Hi @Tallyb - I also couldn't reproduce the log files issue (they do not appear at all for me), but what happened to @GiladShoham happened to me too. We spoke offline and found that this comes from bit having trouble serializing a circular error. Gilad is working on it.

**Comment by GiladShoham** [comment]
solved in 14.4.1
